bazel/repository_locations.bzl

```diff
@@ -433,9 +433,9 @@ DEPENDENCY_REPOSITORIES = dict(
         cpe = "N/A",
     ),
     com_googlesource_code_re2 = dict(
-        sha256 = "04ee2aaebaa5038554683329afc494e684c30f82f2a1e47eb62450e59338f84d",
-        strip_prefix = "re2-2020-03-03",
-        urls = ["https://github.com/google/re2/archive/2020-03-03.tar.gz"],
+        sha256 = "455bcacd2b94fca8897decd81172c5a93e5303ea0e5816b410877c51d6179ffb",
+        strip_prefix = "re2-2b25567a8ee3b6e97c3cd05d616f296756c52759",
+        urls = ["https://github.com/google/re2/archive/2b25567a8ee3b6e97c3cd05d616f296756c52759.tar.gz"],
```

Comment:
Can you please add the date for this commit above this line?

Reply:
done

Reply:
And a TODO to switch back to a release version when your commit is inluded. @junyer generally releases monthly on the first of the month.

Reply:
added a TODO comment.